index.html

```diff
@@ -466,10 +466,10 @@
       automatically generated by the application server or an optimization
       proxy (e.g. a CDN).</p>
       <pre class="example">
-  Link: &lt;https://example.com/font.woff&gt;; rel=preload; as=font
+  Link: &lt;https://example.com/font.woff2&gt;; rel=preload; as=font; type='font/woff2'
   Link: &lt;https://example.com/app/script.js&gt;; rel=preload; as=script
   Link: &lt;https://example.com/logo-hires.jpg&gt;; rel=preload; as=image
-  Link: &lt;https://fonts.example.com/font.woff&gt;; rel=preload; as=font; crossorigin
+  Link: &lt;https://fonts.example.com/font.woff2&gt;; rel=preload; as=font; crossorigin; type='font/woff2'
```

Comment:
`crossorigin` is missing, which prevents your fonts from being properly reused. `nopush` will help you avoid CloudFlare from trying to push these resources down the wrong connection (as fonts are credential-less fetches).

This is not a spec issue, so please open a crbug if you have any further comments/questions.